sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb

```diff
@@ -38,22 +38,22 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    device_fleet_role_arn = ":role/SageMaker*&gt;"
</pre><p>Indicate to the customer that they need to supply this ARN.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Sure